userscript.js

```diff
@@ -70,9 +74,9 @@
     var bettered = false;
 
     var runBetterGoogle = function() {
-        if (prevResultCount != document.querySelectorAll('.g .yuRUbf').length) {
+        if (prevResultCount != document.querySelectorAll('.g .yuRUbf .rnBE4e').length) {
             document.querySelectorAll('.g .yuRUbf').forEach(betterGoogleRow);
-            prevResultCount = document.querySelectorAll('.g .yuRUbf').length;
+            prevResultCount = document.querySelectorAll('.g .yuRUbf .rnBE4e').length;
```

Comment:
I'm now getting a different version of the "More options" button that has the `csDOgf` class and does some weird stuff with absolute positioning. I'm going to try without this hunk locally and see if things still work fine.